src/lib/store.js

```diff
@@ -13,7 +14,12 @@ const loggerMiddleware = createLogger()
 
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose
 
-const middlewares = [thunkMiddleware, loggerMiddleware]
+const middlewares = [
+  thunkMiddleware,
+  flag('store.redux-logger') && process.env.NODE_ENV !== 'browser:production'
+    ? loggerMiddleware
+    : null
+].filter(Boolean)
```

Comment:
this is not a `chore` commit since you are changing the code we use in production ;)

Why this test on NODE_ENV?